packages/number-field/README.md

```diff
@@ -230,11 +265,103 @@ If the user types a value that is between two steps and blurs the input, the val
 ></sp-number-field>
 ```
 
-## Default value
+</sp-tab-panel>
+</sp-tabs>
+
+### States
```

Comment:
Looks like there's maybe a duplicate `States` section at the bottom. Unless that's supposed to be wrapped into the accessibility section above it? 

<img width="410" alt="Screenshot 2025-04-30 at 12 19 50 PM" src="https://github.com/user-attachments/assets/2ca1858d-5d4a-49ce-8c68-05fe2f8b50be" />